websockets/Create-http-urls.any.js

```diff
@@ -0,0 +1,19 @@
+test(() => {
+  const url = new URL ("/", location);
+  url.protocol = "http";
+  const httpURL = url.href;
+  url.protocol = "https";
+  const httpsURL = url.href;
+  url.protocol = "ws";
+  const wsURL = url.href;
+  url.protocol = "wss";
+  const wssURL = url.href;
+
+  let ws = new WebSocket(httpURL);
+  assert_equals(ws.url, wsURL);
+  ws.close();
```

Comment:
If the assertion fails then this is not reached. Maybe it doesn't matter? If it does, use `this.add_cleanup`

Reply:
These have passed review downstream already and addressing this consistently is a bit tricky for several tests. Since it doesn't matter too much I guess I'd rather leave this as-is.

Reply:
OK, fair enough.